contracts/src/SetupHelper.sol

```diff
@@ -15,11 +15,11 @@ contract SetupHelper {
         bool isDayPassTransferble, // NFT or SBT
         uint256 gasLimitPerOperation, // How much gas AA can consume in a user operation
         uint256 spendingLimitPerOperation, // How much token AA can transfer in a user transaction
-        uint256 timeLimitInSecond, // how long the NFT is available
+        uint48 timeLimitInSecond, // how long the NFT is available
```

Comment:
Switching to uint48 to conform with what the paymaster validation function should return